tests/datastore/tests.csv

```diff
@@ -0,0 +1,15 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+before,0,0,q,.servers.startup[],1,,"Start TorQ connection management"
+before,0,0,q,.proc.sys "sleep 2",1,,"Wait for proc to start"
+before,0,0,q,".rdb.ignorelist,:`KUT`KUTR",1,,"Add test result tables to ignore list"
+before,0,0,q,stph:gethandle[`stp1],1,,"Open STP handle"
```

Comment:
I'm not sure this line is required - `.servers.startup[]` should have opened the handle and I don't think you use `stph` later in the tests

```suggestion
```